src/UI/Component/Input/Field/Factory.php

```diff
@@ -350,6 +350,50 @@ public function password($label, $byline = null);
 	public function select($label, array $options, $byline = null);
 
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A textarea is intended for entering multi-line texts.
+	 *   composition: >
+	 *      Textarea fields will render an textarea HTML tag.
+	 *      If a limit is set, a byline about limitation is automatically set.
+	 *   effect: >
+	 *      Textarea inputs are NOT restricted to one line of text.
+	 *      A textarea counts the amount of character input by user and displays the number.
+	 *   rivals:
+	 *      text field: Use a text field if users should input only one line of text.
+	 *      numeric field: Use a numeric field if users should input numbers.
+	 *      alphabet field: >
+	 *          Use an alphabet field if the user should input single letters.
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Textarea Input MUST NOT be used for choosing from predetermined options.
+	 *     2: >
+	 *         Textarea input MUST NOT be used for numeric input, a Numeric Field is
+	 *         to be used instead.
+	 *     3: >
+	 *         Textarea Input MUST NOT be used for letter-only input, an Alphabet Field
+	 *         is to be used instead.
+	 *     4: >
+	 *         Textarea Input MUST NOT be used for single-line input, a Text Field
+	 *         is to be used instead.
+	 *     5: >
+	 *         If a min. or max. number of characters is set for textarea, a byline MUST
+	 *         be added stating the number of min. and/or max. characters.
+	 *   interaction:
+	 *     1: >
+	 *         Textarea Input MAY limit the number of characters, if a certain length
+	 *         of text-input may not be exceeded (e.g. due to database-limitations).
+	 *
+	 * ---
+	 * @param    string      $label
+	 * @param    string|null $byline
+	 * @return    \ILIAS\UI\Component\Input\Field\TextArea
+	 */
+	public function textArea($label, $byline = null);
```

Comment:
@Amstutz Do we really want to say `textArea` instead of `textarea`? To me it feels like one word, thus no capital in the middle.

Reply:
You are right @klees. @xus since you use textarea as one word everywhere else in your text, you should do so here as well, meaning no (lower) camel case, just textarea.